code/game/machinery/slotmachine.dm

```diff
@@ -254,14 +254,15 @@
 /obj/machinery/computer/slot_machine/proc/give_prizes(usrname, mob/user)
 	var/linelength = get_lines()
 
-	if(reels[1][2] + reels[2][2] + reels[3][2] + reels[4][2] + reels[5][2] == "[SEVEN][SEVEN][SEVEN][SEVEN][SEVEN]")
+	if(jackpots >= 0)
+	//if(reels[1][2] + reels[2][2] + reels[3][2] + reels[4][2] + reels[5][2] == "[SEVEN][SEVEN][SEVEN][SEVEN][SEVEN]")
```

Comment:
What's this changed for?

Reply:
oh, I was testing something, sorry. Lemme fix that

Reply:
fixed in my next pull